libbeat/autodiscover/providers/kubernetes/kubernetes.go

```diff
@@ -111,19 +111,22 @@ func (p *Provider) emitEvents(pod *kubernetes.Pod, flag string, containers []kub
 	containerstatuses []kubernetes.PodContainerStatus) {
 	host := pod.Status.PodIP
 
-	// Collect all container IDs
+	// Collect all container IDs and runtimes from status information. Kubernetes has both docker and rkt
```

Comment:
minor thing here, but it could be others at some point (apart from docker and rkt)

Reply:
should i update this?

Reply:
if you want :)

Reply:
done